File: DTOs/Sources/PosterServiceDTOs/PosterPositionShortResponseDTO.swift
import Foundation

public struct PosterPositionShortResponseDTO: Codable {
    public var id: UUID
    public var posterId: UUID?
    public var expiresAt: Date
    public var status: String

    public init(id: UUID, posterId: UUID? = nil, expiresAt: Date, status: String) {
        self.id = id
        self.posterId = posterId
        self.expiresAt = expiresAt
        self.status = status
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/PosterServiceDTOs/PosterPositionShortResponseDTO.kt
package net.ipv64.kivop.dtos.PosterServiceDTOs

import java.time.LocalDateTime
import java.util.UUID

data class PosterPositionShortResponseDTO(
    var id: UUID,
    var posterId: UUID?,
    var expiresAt: LocalDateTime,
    var status: String,
)
